instrumentation/akka/akka-actor-fork-join-2.5/javaagent/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ muzzle {
   pass {
     group.set("com.typesafe.akka")
     module.set("akka-actor_2.11")
-    versions.set("[2.5,)")
+    versions.set("[2.5,2.6)") // Akka's custom ForkJoin was removed in 2.6, replaced by the java.concurrent version
```

Comment:
`akka-actor_2.11` doesn't look like it goes above 2.5.x

maybe instead (or in addition), add

```
pass {
    group.set("com.typesafe.akka")
    module.set("akka-actor_2.12")
    versions.set("[2.5,2.6)") // Akka's custom ForkJoin was removed in 2.6, replaced by the java.concurrent version
    assertInverse.set(true)
}
```

Reply:
Good point! Pushed a new commit with that change.